mono/metadata/loader.c

```diff
@@ -60,7 +60,7 @@
  */
 static MonoCoopMutex loader_mutex;
 static mono_mutex_t global_loader_data_mutex;
-static gboolean loader_lock_inited;
+static gboolean loader_lock_inited = FALSE;
 static gboolean loader_lock_track_ownership = FALSE;
```

Comment:
Static variables don't need to be initialized to NULL/FALSE.

Reply:
Oh whoops, guess I missed that it's static, will fix.

Reply:
Extern ones don't need it either.